Return-Path: <olaf@suse.de>
Received: from hermes.suse.de (Hermes.suse.de [10.0.0.1])
	by wotan.suse.de (Postfix) with ESMTP id EDA9515A1B
	for <olh@wotan.suse.de>; Mon, 16 Feb 2004 21:23:06 +0100 (CET)
Received: by hermes.suse.de (Postfix)
	id EB09311E0D; Mon, 16 Feb 2004 21:23:06 +0100 (CET)
Received: from mandarine.suse.de (Mandarine.suse.de [10.10.3.21])
	by hermes.suse.de (Postfix) with ESMTP
	id E4D47AA13; Mon, 16 Feb 2004 21:23:06 +0100 (CET)
Received: by mandarine.suse.de (Postfix, from userid 90)
	id CF4B0469E7; Mon, 16 Feb 2004 21:23:06 +0100 (CET)
To: mantel@suse.de, olh@suse.de
Subject: Incoming changes to linux-2.5 cset 1.1695  mandarine.suse.de:/olaf/sources/tree/linux-2.5 
Message-Id: <20040216202306.CF4B0469E7@mandarine.suse.de>
Date: Mon, 16 Feb 2004 21:23:06 +0100 (CET)
From: olaf@suse.de (Olaf Hering)
X-Spam-Checker-Version: SpamAssassin 2.63 (2004-01-11) on wotan.suse.de
X-Spam-Level: 
X-Spam-Status: No, hits=-4.8 required=5.0 tests=AWL,BAYES_00 autolearn=ham 
	version=2.63
Content-Length: 1500

old status OK
ChangeSet
  1.1695 04/02/16 10:00:39 anton@samba.org +1 -0
  [PATCH] fix ppc64 LPAR
  
  This fixes pSeries LPAR (logical partitioned) machines.  We weren't
  initialising the pci_dma_ops stuff.

  arch/ppc64/kernel/pSeries_lpar.c
    1.29 04/02/15 23:30:46 anton@samba.org[torvalds] +2 -0
    fix ppc64 LPAR

.........................................................................
# This is a BitKeeper generated patch for the following project:
# Project Name: Linux kernel tree
# This patch format is intended for GNU patch command version 2.5 or higher.
# This patch includes the following deltas:
#	           ChangeSet	1.1694  -> 1.1695 
#	arch/ppc64/kernel/pSeries_lpar.c	1.28    -> 1.29   
#
# The following is the BitKeeper ChangeSet Log
# --------------------------------------------
# 04/02/16	anton@samba.org	1.1695
# [PATCH] fix ppc64 LPAR
# 
# This fixes pSeries LPAR (logical partitioned) machines.  We weren't
# initialising the pci_dma_ops stuff.
# --------------------------------------------
#
diff -Nru a/arch/ppc64/kernel/pSeries_lpar.c b/arch/ppc64/kernel/pSeries_lpar.c
--- a/arch/ppc64/kernel/pSeries_lpar.c	Mon Feb 16 21:23:05 2004
+++ b/arch/ppc64/kernel/pSeries_lpar.c	Mon Feb 16 21:23:05 2004
@@ -296,6 +296,8 @@
 {
 	pSeries_lpar_mm_init();
 
+	tce_init_pSeries();
+
 	ppc_md.tce_build	 = tce_build_pSeriesLP;
 	ppc_md.tce_free_one	 = tce_free_one_pSeriesLP;
 
.........................................................................
# vim: syntax=diff

